conda-forge.yml

```diff
@@ -9,5 +9,9 @@ github:
   branch_name: main
   tooling_branch_name: main
 provider:
+  linux_aarch64: default
+  # Could NOT find Python3 (missing: Python3_INCLUDE_DIRS ...
+  # so skip for now. Also, can't natively test.
```

Comment:
this comment will be deleted on a future migration and bot lint. maybe add `skip: true   # [ppc64le]` and the comment to the recipe.